prepare.sh

```diff
@@ -31,7 +31,7 @@ while true; do
         echo "--gitrepo                 specify the git repository" 
         echo " "
         echo "Sample Usage:"
-        echo "./prepare.sh --gitrepo=https://github.com/example --workdir=/opt/build --ref=pull/123/head"
+        echo "./prepare.sh --gitrepo=https://github.com/example --workdir=tip/ --ref=pull/123/head"
```

Comment:
this can be a separate commit. One for the corrections to the prepare.sh script, one for the update to the groovy job

Reply:
Thanks, I'll make the required changes